api/proto/teleport/legacy/client/proto/event.proto

```diff
@@ -225,5 +226,6 @@ message Event {
     teleport.scopes.access.v1.ScopedRole ScopedRole = 80;
     // ScopedRoleAssignment is an assignment of one or more scoped roles to a user.
     teleport.scopes.access.v1.ScopedRoleAssignment ScopedRoleAssignment = 81;
+    teleport.presence.v1.RelayServer relay_server = 82;
```

Comment:
should we add a godoc here for consistency lol

`// relay_server is a relay server` 😆